main/ertp/ERTP-Introduction.md

```diff
@@ -0,0 +1,117 @@
+# ETRP Introduction
+
+ERTP (*Electronic Rights Transfer Protocol*) is Agoric's token
+standard for transferring tokens and other digital assets in
+JavaScript. Using the ETRP API, you can easily create digital assets,
+all of which are transferred exactly the same way and with exactly the
+same security properties. 
+
+ERTP uses object capabilities to enforce access control. If your
+program has a reference to an object, it can call methods on that
+object. If it doesn't have a reference, it can't. 
+
+## Creating assets with ERTP
+
+In ERTP, *mints* create digital *assets*. Access to an asset type's
+mint lets you create more digital assets of that type. You can then
+store new assets in a *payment* or a *purse*. 
+- **Payments**: Assets you intend to trade with someone else.
```

Comment:
```suggestion
- **Payments**: Assets you intend to send to someone else.
```
They don't have to be traded, although that's probably a likely case. If we only say trade, it sounds like we're excluding the use case that people would expect.